helpdesk_mgmt/security/ir.model.access.csv

```diff
@@ -2,10 +2,10 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_helpdesk_ticket_manager,helpdesk.ticket.manager,model_helpdesk_ticket,group_helpdesk_manager,1,1,1,1
 access_helpdesk_ticket_user,helpdesk.ticket.user,model_helpdesk_ticket,group_helpdesk_user,1,1,1,0
 access_helpdesk_ticket_user_personal,helpdesk.ticket.user.personal,model_helpdesk_ticket,group_helpdesk_user_own,1,1,1,0
-access_helpdesk_ticket_portal,helpdesk.ticket.portal,model_helpdesk_ticket,base.group_portal,1,0,0,0
+access_helpdesk_ticket_portal,helpdesk.ticket.portal,model_helpdesk_ticket,base.group_portal,1,1,1,0
```

Comment:
Don't do this! You are giving access to any ticket for being modified to a portal user for example using XML-RPC. Put the controller with sudo when creating the ticket instead.

Reply:
I fixed already your request change. Sorry for the delay.
I will be waiting for any other changes required.

Thank you Pedro.